src/runner.js

```diff
@@ -183,7 +183,7 @@ assign(Runner.prototype, {
   ensureConnection() {
     return Promise.try(() => {
       return this.connection || new Promise((resolver, rejecter) => {
-        this.client.acquireConnection()
+        return this.client.acquireConnection()
```

Comment:
Okay...this seems like a best practice though and we aren't explicitly calling it out elsewhere? I can certainly add the comments